CMakeLists.txt

```diff
@@ -34,3 +34,10 @@ if(BUILD_DD_EVAL_TESTS)
   include(GoogleTest)
   add_subdirectory(test)
 endif()
+
+option(BUILD_DD_EVAL_BENCHMARKS "Also build benchmarks for DDEval project")
+if(BUILD_DD_EVAL_BENCHMARKS)
+  enable_testing()
+  include(GoogleTest)
+  add_subdirectory(benchmarks)
+endif()
```

Comment:
If you want the cpp-linter to work properly, you also have to set that option in the corresponding CI workflow (`.github/workflows/cpp-linter.yaml`)